README.md

```diff
@@ -14,8 +14,10 @@ This crate provides utilities which make it easy to perform zero-copy
 parsing and serialization by allowing zero-copy conversion to/from byte
 slices.
 
-This is enabled by three core marker traits, each of which can be derived
+This is enabled by four core marker traits, each of which can be derived
 (e.g., `#[derive(FromBytes)]`):
+- `FromZeroes` indicates that a sequence of zero bytes represents a valid
```

Comment:
This wording is a little awkward, it could be interpreted as a sequence of length 0. Maybe something like "`FromZeroes` indicates that a sequence of bytes all set to zero represents a valid instance of a type.

Reply:
Good call; I've updated it, but kept the "Types..." prefix to be consistent with other traits. I can change all of them if you'd prefer.

Reply:
Missed change? I don't see this in the latest revision.

Reply:
Here's the diff I'm looking at: https://github.com/google/zerocopy/compare/afcea57c53372fd04d28bca8b2ea90db429a86e6..3fb2a0e342304d60ca354102d8bc2d7d2181d6fc